library/general/src/modules/PackagesProposal.rb

```diff
@@ -219,10 +220,11 @@ def AddTaboos(unique_id, resolvables)
     # @param [String] unique_id the unique identificator
     # @param [Array<String>] resolvables list of resolvables to taboo
     # @return [Boolean] whether successful
-    def SetTaboos(unique_id, resolvables)
+    def SetTaboos(unique_id, type, resolvables)
```

Comment:
same here for missing type documentation. Also missing verify type in CheckParams....and last but not least. Is valid to pass nil resolvables to remove taboos?

Reply:
No, you should pass an empty array to remove all of them.